python/tests/remote/test_remote_platform.py

```diff
@@ -19,81 +21,120 @@ def startUpMockServer():
     cudaq.reset_target()
 
 
+skipRemoteBackendForEagerMode = pytest.mark.skipif(
+    os.getenv("CUDAQ_PYTEST_EAGER_MODE") == 'ON',
+    reason="remote qpu supported only for MLIR mode of execution")
+
+
+@pytest.fixture(autouse=True)
+def do_something():
+    yield
+    cudaq.__clearKernelRegistries()
+
+
 def test_setup():
     target = cudaq.get_target()
     numQpus = target.num_qpus()
     assert numQpus == num_qpus
 
 
+def check_sample(entity):
+
+    def check_basic(counts):
+        assert len(counts) == 2
+        assert "00" in counts
+        assert "11" in counts
```

Comment:
assert 01 and 10 are not in counts?

Reply:
I think that is covered by len(counts) == 2